src/main/resources/application-demo.yaml

```
@@ -13,3 +13,5 @@ form-flow:
       gcc:
         prefix: DEMO-
         creation-point: creation
+il-gcc:
+  validate-provider-id: false
```

Carl used `test-main` for acceptance testing before it was closed.  `allow-provider-registration-flow` flag needs to be on to do acceptance testing on Heroku branches.

```suggestion
    validate-provider-id: ${VALIDATE_PROVIDER_ID_FLAG:false}
    allow-provider-registration-flow: ${ALLOW_PROVIDER_REGISTRATION_FLOW:true}
```